test/integration/pack/test_pack.py

```diff
@@ -1075,6 +1075,48 @@ def test_pack_rpm_use_docker(tt_cmd, tmpdir):
     assert rc == 0
 
 
+@pytest.mark.slow
+def test_pack_rpm_use_docker_tnt_version(tt_cmd, tmpdir):
```

Comment:
Add tests for incorrect version format, and the version that does not exist.

Reply:
Added new tests.